flake.nix

```diff
@@ -274,7 +288,10 @@
           toml2json
           llvmPackages.lldb # This ensures the right lldb is in the environment for running rust-lldb
           wasm-bindgen-cli
+          binaryen
           jq
+          rust-bin.stable."1.66.1".default
+          rust-analyzer
```

Comment:
@kobyhallx Why was this added to this PR? Not sure why we'd need to modify the CLI nix config as this PR only addresses the new ABI package.

Reply:
These allow for easier local dev env setup.